src/lightning_app/components/__init__.py

```diff
@@ -1,6 +1,6 @@
 from lightning_app.components.database.client import DatabaseClient
 from lightning_app.components.database.server import Database
-from lightning_app.components.multi_node import MultiNode
+from lightning_app.components.multi_node.multi_node import MultiNode
```

Comment:
Let's add `MultiNode` to the `multi_node/__init__.py`

Reply:
Done !